_test/general.test.php

```diff
@@ -5,13 +5,68 @@
  * @group plugin_findologicxmlexport
  * @group plugins
  */
-class general_plugin_findologicxmlexport_test extends DokuWikiTest {
+
+require_once(__DIR__ . '/../DokuwikiXMLExport.php');
+require_once(__DIR__ . '/../admin.php');
+
+class general_plugin_findologicxmlexport_test extends DokuWikiTest
+{
+    /**
+     * A page needs to have content. This is the placeholder for this value.
+     */
+    const PAGE_CONTENT_PLACEHOLDER = 'This page is for test purposes.';
+
+    /**
+     * Helper function to save a page.
+     *
+     * @param array $ids An array with page ids (eg. start or wiki:syntax)
+     * @param string $content Optional content, else the default test value will be set
+     */
+    private function savePages($ids, $content = self::PAGE_CONTENT_PLACEHOLDER)
+    {
+        foreach ($ids as $id) {
+            saveWikiText($id, $content, '');
+            idx_addPage($id);
+        }
+    }
+
+    /**
+     * Gets parameters, calls the Export and returns the XML as SimpleXMLElement.
+     *
+     * @param int $start Optional start value, default 0 will be set
```

Comment:
I believe PHPStorm will be able to figure out the default value from the method declaration. To verify this place your cursor on the method name and press `STRG+Q`.

Reply:
Yes, removing it from PHPDoc